.travis.yml

```diff
@@ -31,13 +40,22 @@ cache:
 
 install:
   - ./.ci_scripts/start_docker.sh
-  - ./.ci_scripts/install_sdl.sh
+  - if [ "$BUILD_TARGET" != "appimage" ];
+    then
+      ./.ci_scripts/install_sdl.sh;
+    else
+      sudo apt-get -y install libgl1-mesa-dev libsdl2-dev libsdl2-mixer-dev;
```

Comment:
Can we move this logic to the `install_sdl.sh` script?

Reply:
Logic has been moved, remove the `if` statement please :)

Reply:
Nice catch!